### PR TITLE
[Inflector] Fix testPluralize() arguments names

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

This is minor but the testPluralize() method arguments names are inversed ($plural contains the singular, $singular contains the plural, etc.). I lost some time on this because I thought the test was wrong 😅 I guess it's better to have a test that is easier to understand.

### CODE REVIEW

File: src/Symfony/Component/Inflector/Tests/InflectorTest.php [hunk 1]
```diff
@@ -294,30 +294,30 @@ public function pluralizeProvider()
     /**
      * @dataProvider singularizeProvider
      */
-    public function testSingularize($plural, $singular)
+    public function testSingularize($plural, $expectedSingular)
```
Comment (fancyweb) [inline src/Symfony/Component/Inflector/Tests/InflectorTest.php:297]: Should I revert the changes on this method or backport it on 3.4?

Comment (nicolas-grekas) [reply src/Symfony/Component/Inflector/Tests/InflectorTest.php:297]: yes please

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fancyweb.
